src/encoded/static/components/item-pages/BiosourceView.js

```diff
@@ -20,7 +20,7 @@ export default class BiosourceView extends DefaultItemView {
 
         initTabs.push(ExperimentSetsTableTabView.getTabObject(_.extend({}, this.props, {
             'searchHref' : (
-                "/search/?type=ExperimentSetReplicate&experimentset_type=replicate&" +
+                "/browse/?type=ExperimentSetReplicate&experimentset_type=replicate&" +
```

Comment:
Was there a reason for this change?
(Looks good/fine, just slightly curious if response from URL is different in some way that matters like columns).

Reply:
It's been a while, but I remember talking to Burak that it should look the same as BrowseView. That's why we updated.

Reply:
Seems is set to /browse/ here so we navigate to /browse/ when click on btn (which is great approach IMO).